wrappers/javascript/anoncreds-nodejs/src/NodeJSAnoncreds.ts

```diff
@@ -172,8 +171,8 @@ export class NodeJSAnoncreds implements Anoncreds {
       attributeRawValues as unknown as Buffer,
       attributeEncodedValues as unknown as Buffer,
       revocationRegistryId,
-      revocationStatusList,
-      revocationConfiguration.ref(),
+      options.revocationStatusList?.handle ?? 0,
```

Comment:
Isn't this handled in the `serializeArguments` function? I would strongly prefer not to send the `0` as that can be a valid handle number. 

If `serializeArguments` does not handle this we should 100% do it there. Maybe we should just use `USIZE::MAX`?

Reply:
The problem with `serializeArguments` is that it receives the object as undefined so it does not know which type of object it is supposed to be (i.e. typeof arg = 'undefined'). So it returns NULL, which works fine for optional strings but not for optional object handles (whose type is `size_t`).